### PR TITLE
Firstpost

### PR DESCRIPTION
**Pull request by fern4lvarez**

### CODE REVIEW

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+date: 2015-03-14 00:00:00
+description: We are thrilled to announce our existence, so hi, here we are!
+author: oscillatingworks
+hn-discussion:
+---
+
+Hi everyone!
+
+This is **[oscillatingworks]**, and we love open source. Our mission is to find
+exciting independent, homemade open source software projects, work hand by hand
```
Comment (mkorszun) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: comma after exciting and homemade?

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  them using our **[compass]**.
+- We discuss the adequacy of each submitted project and take a decision on which
+  could be its next **[phase]**. 
+- In case we decide to keep on working on a project, we spend a considerable amount
+  of time on understanding how it works from a technical point of view, and finding
+  its weakest points, so we know the kind of work that it requires.
+- We keep in touch with the author and tell them our contribution plans, and how they
+  fit into their own plans as an author. Depending on their feedback, we start actively
+  working on the project.
+- Once we become official maintainers of the project, we take care that they it enjoys
```
Comment (mkorszun) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: remove 'they' -> we take care that it enjoys

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  a healthy lifetime: we handle issues, publish new releases, review and merge pull
+  requests, fix bugs, etc. 
+
+Our works are not limited to technical aspects, we want to make projects more usable
```
Comment (mkorszun) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: 1. Our work is not
   or
2. Our tasks are not

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md (hunk 2 above)
Comment (fern4lvarez) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: ah yes

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md (hunk 3 above)
Comment (fern4lvarez) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: Like num. 2

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md (hunk 1 above)
Comment (fern4lvarez) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: I'll go with `exciting, independent, homemade open source software projects`

**Comment by mkorszun** [comment]
@fern4lvarez so can I merge it?

**Comment by fern4lvarez** [comment]
I'll squash and then let you know.

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Our action plan use to look like this:
+
+- We keep track of small and attractive indie open source projects and report on
+  them using our **[compass]**.
+- We discuss the adequacy of each submitted project and take a decision on which
+  could be its next **[phase]**.
+- In case we decide to keep on working on a project, we spend a considerable amount
+  of time on understanding how it works from a technical point of view, and finding
+  its weakest points, so we know the kind of work that it requires.
+- We keep in touch with the author and tell them our contribution plans, and how they
```
Comment (mkorszun) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: author - > authors

File: _posts/2015-03-14-Hi-this-is-oscillatingworks.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  fit into their own plans as an author. Depending on their feedback, we start actively
```
Comment (mkorszun) [inline _posts/2015-03-14-Hi-this-is-oscillatingworks.md]: fit into author plans.

**Comment by fern4lvarez** [comment]
Updated.

**Pull request merged**